src/cryptonote_basic/cryptonote_basic_impl.h

```diff
@@ -98,31 +107,14 @@ namespace cryptonote {
     , const crypto::hash8& payment_id
     );
 
-  bool get_account_integrated_address_from_str(
-      account_public_address& adr
-    , bool& has_payment_id
-    , crypto::hash8& payment_id
-    , bool testnet
-    , const std::string& str
-    );
-
   bool get_account_address_from_str(
```

Comment:
The build machines get this:

```
build/src/cryptonote_basic/cryptonote_basic_impl.h:110:8: note: in passing argument 1 of 'bool cryptonote::get_account_address_from_str(cryptonote::address_parse_info&, bool, const string&)'
   bool get_account_address_from_str(
        ^
build/src/rpc/daemon_handler.cpp: In member function 'void cryptonote::rpc::DaemonHandler::handle(const cryptonote::rpc::MiningStatus::Request&, cryptonote::rpc::MiningStatus::Response&)':
build/src/rpc/daemon_handler.cpp:521:80: error: cannot convert 'const cryptonote::account_public_address' to 'bool' for argument '2' to 'std::__cxx11::string cryptonote::get_account_address_as_str(bool, bool, const cryptonote::account_public_address&)'
       res.address = get_account_address_as_str(m_core.get_testnet(), lMiningAdr);
                                                                                ^
```

Reply:
Thank you for pointing it out! Fixed now.